### PR TITLE
Assorted spelling and grammar fixes.

### PR DESCRIPTION
**Pull request by CapnKernel**

### CODE REVIEW

File: pages/blog/session-auth-spa.yml [hunk 1]
```diff
@@ -34,15 +34,15 @@ If you're using `fetch` you will need to set `credentials` to at least
 Django's session-based auth works by storing the user ID and the auth backend
 that found them in the session.
 
-So long as the user agent (e.g. your browser) sends the cookie, it will "just
+So, as long as the user agent (e.g. your browser) sends the cookie, it will "just
```
Comment (funkybob) [inline pages/blog/session-auth-spa.yml]: Please revert.

File: pages/blog/session-auth-spa.yml [hunk 2]
```diff
@@ -232,6 +232,6 @@ comment and let me know, and I'll update this post!
 
 ## Thanks
 
-Many thanks to Bjorn Dissing who prompted me to finally write this post, and
+Many thanks to Bjørn Dissing who prompted me to finally write this post, and
```
Comment (funkybob) [inline pages/blog/session-auth-spa.yml:235]: Can you show evidence this is how they spell their own name?

File: pages/blog/tokens-aint-tokens.yml [hunk 3]
```diff
@@ -94,43 +94,43 @@ Shared Secret systems involve the Issuer and Consumers having a copy of the
 same key.  Typically this uses a system like
 [HMAC](https://en.wikipedia.org/wiki/HMAC).
 
-Without knowing the secret you can not generate or verify a token.
+ You cannot generate or verify a token without knowing the secret.
```
Comment (funkybob) [inline pages/blog/tokens-aint-tokens.yml:97]: I'd prefer to keep the condition first in the statement.

File: pages/blog/tokens-aint-tokens.yml [hunk 4]
```diff
@@ -94,43 +94,43 @@ Shared Secret systems involve the Issuer and Consumers having a copy of the
...
+to have the same key, making them _all_ hacking targets.
 
 This means the Issuer *must* trust *every* Consumer not to abuse the Shared
 Secret to generate their own tokens.
 
 [Public Key
 cryptography](https://en.wikipedia.org/wiki/Public-key_cryptography) allows you
 to safely publicise the Public Key (used for decrypting, and verifying
-signatures) whilst carefully guarding the Private Key (used for encrypting, and
+signatures) while carefully guarding the Private Key (used for encrypting, and
```
Comment (funkybob) [inline pages/blog/tokens-aint-tokens.yml:108]: How is whilst incorrect here?

File: pages/blog/tokens-aint-tokens.yml [hunk 5]
```diff
@@ -94,43 +94,43 @@ Shared Secret systems involve the Issuer and Consumers having a copy of the
...
 signing).
 
-This also means none of the Consumers can generate a valid token.
+This also means Consumers cannot generate valid tokens.
```
Comment (funkybob) [inline pages/blog/tokens-aint-tokens.yml:111]: Explain your reasoning for this rephrasing.

File: pages/blog/tokens-aint-tokens.yml [hunk 6]
```diff
@@ -94,43 +94,43 @@ Shared Secret systems involve the Issuer and Consumers having a copy of the
...
 
 ## Which to use?
 
 If your system is monolithic, simple opaque tokens can suffice. An example of
-this is Django's session auth, if you treat the Session Cookie value as the
+this is Django's session auth, where the Session Cookie value is treated as the
```
Comment (funkybob) [inline pages/blog/tokens-aint-tokens.yml:116]: s/is treated/can be considered/

File: pages/blog/tokens-aint-tokens.yml [hunk 7]
```diff
@@ -94,43 +94,43 @@ Shared Secret systems involve the Issuer and Consumers having a copy of the
...
 
-They can also continue to work as Opaque tokens when simply posessing the token
+They can also continue to work as Opaque tokens when simply possessing the token
 is sufficient authorisation to access the resource.
 
 If your system is distributed, but private (all Consumers are under your
-control) you might consider using a Shared Secret signed token, as you can more
-readily trust your Consumers to not maliciously issue tokens.
+control) you might consider using a Shared Secret signed token, as you can be more
+confident that your Consumers won't maliciously issue tokens.
```
Comment (funkybob) [inline pages/blog/tokens-aint-tokens.yml:133]: I prefer the original wording.

File: pages/blog/tokens-aint-tokens.yml [hunk 8]
```diff
@@ -142,26 +142,26 @@ Most combinations of the above have been tried over the years, and
...
 
-1. Oauth2
+1. OAuth2
 
     Originally this used opaque tokens, since the Issuer is also the Consumer.
 
 1. JWT
 
-    Commonly, JWT are Parseable and Signed.
+    Commonly, JWT tokens are Parsable and Signed.
```
Comment (funkybob) [inline pages/blog/tokens-aint-tokens.yml:154]: Since the T in JWT is Token... adding tokens here is redundant.

**Comment by CapnKernel** [comment]
Hi Curtis,

I appreciate you sending your pull request comments.  I'm providing it as
is.  Take from it whatever you like, from nothing to everything.

Regarding Bjørn's name, the github page you gave is:

  https://github.com/bsdis

One of Bjørn's repos is:

  https://github.com/bsdis/healthyhouse-api

blob/master/AUTHORS.rst file says:

Development Lead: Healthyhouse ***@***.***>

And I found this, which says Bjørn Dissing is the owner of Radonmeters.com


https://steelkiwi.com/projects/an-online-service-that-provides-full-cycle-indoor-radon-measurements/

And this company filing, which says Bjørn Skovlund Dissing as the director
of Healthyhouse (so bsdis makes sense):

  https://cvrapi.dk/virksomhed/dk/healthyhouse-aps/38752693

Mitch.

>

**Comment by funkybob** [comment]
Please don't take my comments as disrespect. I truly appreciate that anyone took that much time to review and edit my posts :)

Almost all of what you've provided I'll be delighted to accept as is.

A couple of cases are personal preference... as for Bjørn ... I looked quickly at their repo, and didn't see their name written. I didn't want to assume one way or the other.

**Comment by funkybob** [comment]
Unfortunately, github doesn't let me cherry pick which edits I commit. So I was hoping to work with you to resolve the differences so I could merge this PR.

**Comment by CapnKernel** [comment]
Hi Curtis,

On Wed, Jul 21, 2021 at 2:19 PM Curtis Maloney ***@***.***>
wrote:

> Please don't take my comments as disrespect.
>

None taken.  What you've done is standard practice for pull requests.  I
think the way you've done it is particularly important when considering
contributions to an open source project: asking the contributor to go back
and make the changes helps the contributor learn more about the way things
are done in a particular project.

> A couple of cases are personal preference... as for Bjørn ... I looked
> quickly at their repo, and didn't see their name written. I didn't want to
> assume one way or the other.
>
I felt the same at the time.

github doesn't let me cherry pick which edits I commit


I understand.  If I was in your situation and wanted to cherry pick, I'd
download it as a diff and apply it with patch to a checked out repo.  I'd
then only commit the changes I wanted.  Alternatively, it may be possible
to accept the pull request on github into a branch.  You could then pull
that branch and use a merging tool to grab what you wanted.  (I use meld)

I really don't mind if the effort for you means you don't take the
changes.  But I'm handing it to you as-is.

My warmest regards,

Mitch.

>

**Comment by funkybob** [comment]
I've manually merged in your corrections.

Many thanks for taking the time!

**Pull request closed without merging**